src/librustdoc/templates/page.html

```diff
@@ -127,7 +127,7 @@ <h2 class="location"></h2>
                                     type="search"> {#- -#}
                             </div> {#- -#}
                             <button type="button" id="help-button" title="help">?</button> {#- -#}
-                            <a id="settings-menu" href="{{page.root_path|safe}}settings.html" title="settings"> {#- -#}
+                            <a id="settings-menu" href="javascript:void(0)" title="settings"> {#- -#}
```

Comment:
We should turn this into a button, like the help button. Then we don't need the href. That will also provide better hints to audio navigation about its role.

Reply:
:+1:

Reply:
If `settings.html` is going to continue to exist, shouldn’t this still link to it, so that you can right-click on it and open it in a new tab?

Reply:
Good point. I retract my previous feedback about turning it into a button.

Reply:
So I need to catch the URL change?

Reply:
I’m pretty sure you’d want to:

* Leave this thing as a link pointing at settings.html.
* Add an onclick event handler that cancels the default behavior, using pushstate and DOM manipulation to imitate a page change.